Rubberduck.Core/UI/CodeExplorer/Commands/CodeExplorerFindAllReferencesCommand.cs

```diff
@@ -24,6 +24,15 @@ public CodeExplorerFindAllReferencesCommand(RubberduckParserState state, FindAll
         {
             _state = state;
             _finder = finder;
+
+            AddToCanExecuteEvaluation(SpecialEvaluateCanExecute);
+        }
+
+        private bool SpecialEvaluateCanExecute(object parameter)
+        {
+            return ((ICodeExplorerNode)parameter).Declaration != null &&
+                   (!(parameter is CodeExplorerReferenceViewModel reference) || !reference.IsDimmed) &&
```

Comment:
could we de-morgan this into `!(parameter is CodeExplorereReferenceViewModel reference && reference.IsDimmed)`? Or am I missing something because I can't think straight?

Reply:
Also I think we should pull the parser-state check to an earlier position, that is the cheapest check and it probably applies a lot...

Reply:
TBH, I have blindly copied from the original `EvaluateCanExecute`.

I will adjust this.